projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -201,43 +218,35 @@ <h3>{{ altgr_title }}</h3>
             </ng-container>
 
             <!-- NOTES -->
-            <ng-container *ngIf="record.metadata.note && record.metadata.note.length > 0 && record.metadata.note | notesFormat as notes">
-              <ng-container *ngIf="notes.general">
-                <dt class="col-sm-4 offset-sm-2 offset-md-0" translate>
-                  Note
-                </dt>
-                <dd [ngClass]="ddCssClass">
-                  <ul class="list-unstyled mb-0">
-                    <li
-                      *ngFor="let note of notes.general"
-                    >{{ note }}</li>
-                  </ul>
-                </dd>
-              </ng-container>
-              <ng-container *ngIf="notes.otherPhysicalDetails">
-                <dt class="col-sm-4 offset-sm-2 offset-md-0" translate>
-                  Physical details
-                </dt>
-                <dd [ngClass]="ddCssClass">
-                  <ul class="list-unstyled mb-0">
-                    <li
-                      *ngFor="let note of notes.otherPhysicalDetails"
-                    >{{ note }}</li>
-                  </ul>
-                </dd>
-              </ng-container>
-              <ng-container *ngIf="notes.accompanyingMaterial">
-                <dt class="col-sm-4 offset-sm-2 offset-md-0" translate>
-                  Accompanying material
-                </dt>
-                <dd [ngClass]="ddCssClass">
-                  <ul class="list-unstyled mb-0">
-                    <li
-                      *ngFor="let note of notes.accompanyingMaterial"
-                    >{{ note }}</li>
-                  </ul>
-                </dd>
-              </ng-container>
+            <ng-container *ngIf="record.metadata.notes_text && record.metadata.notes_text.general">
```

Comment:
`notes_text` doesn't exist any more after the quick fix I merged recently. The correct way to display the notes is the pipe `formatNotes`.

Reply:
I made a mistake with a conflict resolution during a rebase. I hope that it is the only one...